cssom-view-1/Overview.bs

```diff
@@ -1572,10 +1570,7 @@ means to run these steps:
         abort any remaining iteration of this loop.
 1. Let <var>scrollPromise</var> be a new promise.
 1. Return <var>scrollPromise</var>, and run the remaining steps <a>in parallel</a>.
-1. Wait until either all promises in <var>ancestorPromises</var> get resolved with true, or any of them settles in any other state (i.e. gets resolved with false or gets rejected).
-1. Resolve <var>scrollPromise</var>.
-1. Resolve with false any promise in <var>ancestorPromises</var> that is still in the pending state.
-
+1. Resolve <var>scrollPromise</var> when all promises in <var>ancestorPromises</var> have been resolved.
```

Comment:
Maybe `s/have been resolved/have settled`? not sure if we want to ever reject one of these promises, but that seems nicer? Then again that would eat the rejection. So it's fine.